Label     label
GraspPose grasppose
Configuration1D configuration1d
float64   grasp_width